--- a/drivers/lora/native/sx126x/sx126x.c
+++ b/drivers/lora/native/sx126x/sx126x.c
@@ -1516,7 +1516,7 @@
 										\
 	static const struct sx126x_hal_config sx126x_config_##inst = {		\
 		.spi = SPI_DT_SPEC_INST_GET(inst,				\
-					    SPI_WORD_SET(8) | SPI_TRANSFER_MSB), \
+					    SPI_WORD_SET(8) | SPI_TRANSFER_MSB, 0), \
 		.reset = GPIO_DT_SPEC_INST_GET(inst, reset_gpios),		\
 		.busy = GPIO_DT_SPEC_INST_GET(inst, busy_gpios),		\
 		.dio1 = GPIO_DT_SPEC_INST_GET(inst, dio1_gpios),		\
@@ -1571,7 +1571,7 @@
 										\
 	static const struct sx126x_hal_config sx126x_stm32wl_config_##inst = {	\
 		.spi = SPI_DT_SPEC_INST_GET(inst,				\
-					    SPI_WORD_SET(8) | SPI_TRANSFER_MSB), \
+					    SPI_WORD_SET(8) | SPI_TRANSFER_MSB, 0), \
 		.pa_output = SX126X_STM32WL_PA_OUTPUT(inst),			\
 		.rfo_lp_max_power = DT_INST_PROP(inst, rfo_lp_max_power),	\
 		.rfo_hp_max_power = DT_INST_PROP(inst, rfo_hp_max_power),	\
